Gems/GraphCanvas/Code/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@
 #
 #
 
+if(PAL_TRAIT_BUILD_HOST_TOOLS)
```

Comment:
in general, this flag is for tool and builder variant? unless this gem is purely a tool/builder itself?

Reply:
GraphCanvas is a tool framework, it does not have or need a runtime equivalent